docs/src/getting-started/index.md

```diff
@@ -1,10 +1 @@
 # Getting Started
-
-To get started with Forc and Sway start contract development, install the Sway toolchain and Fuel full node and set up your first project.
-
-- [Installation](./installation.md)
-- [The Sway Toolchain](./sway-toolchain.md)
-- [A Forc Project](./forc_project.md)
-- [Temporary Workarounds](./temporary_workarounds.md)
-- [Differences From Solidity](./solidity_differences.md)
-- [Differences From Rust](./rust_differences.md)
```

Comment:
Please keep the one-or-two sentence summary (optional) and ToC for each `index.md` file in each chapter.